pkgs/tools/audio/accuraterip-checksum/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, libsndfile }:
+
+stdenv.mkDerivation rec {
+  name = "accuraterip-checksum-${version}";
+  version = "1.5";
+
+  src = fetchFromGitHub {
+    owner = "leo-bogert";
+    repo = "accuraterip-checksum";
+    rev = "version${version}";
+    sha256 = "1a6biy78jb094rifazn4a2g1dlhryg5q8p8gwj0a60ipl0vfb9bj";
+  };
+
+  buildInputs = [ libsndfile ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D -m755 accuraterip-checksum "$out/bin/accuraterip-checksum"
+
+    runHook postInstall
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A C99 commandline program to compute the AccurateRip checksum of singletrack WAV files";
```

Comment:
I think this description could be simplified to "Computes the AccurateRip checksum of singletrack WAV files".